packages/obojobo-document-engine/src/scripts/oboeditor/plugins/editor-schema.js

```diff
@@ -0,0 +1,57 @@
+import { CHILD_REQUIRED, CHILD_TYPE_INVALID } from 'slate-schema-violations'
+
+// Default node type
+const TEXT_NODE = 'ObojoboDraft.Chunks.Text'
+
+// Placeholder array until EditorStore is implemented
+const nodes = [
+	'ObojoboDraft.Chunks.ActionButton',
+	'ObojoboDraft.Chunks.Break',
+	'ObojoboDraft.Chunks.Code',
+	'ObojoboDraft.Chunks.Figure',
+	'ObojoboDraft.Chunks.Heading',
+	'ObojoboDraft.Chunks.HTML',
+	'ObojoboDraft.Chunks.IFrame',
+	'ObojoboDraft.Chunks.List',
+	'ObojoboDraft.Chunks.MathEquation',
+	'ObojoboDraft.Chunks.MCAssessment',
+	'ObojoboDraft.Chunks.Table',
+	'ObojoboDraft.Chunks.Text',
+	'ObojoboDraft.Chunks.YouTube',
+	'ObojoboDraft.Chunks.QuestionBank',
+	'ObojoboDraft.Chunks.Question',
+	'ObojoboDraft.Sections.Assessment',
+	'ObojoboDraft.Pages.Page'
+]
+
+const matcher = nodes.map(chunk => {
+	return { type: chunk }
+})
+
+const EditorSchema = {
+	schema: {
+		document: {
+			nodes: [{ match: matcher, min: 1 }],
```

Comment:
since matcher doesn't do anything fancy, would probably just opt for 
`nodes: [{ match: nodes.map(n => {type: n}), min: 1 }]`